app/jobs/deposit_work_job.rb

```diff
@@ -97,4 +94,36 @@ def accession?(new_cocina_object:)
     # If work already opened, then the work has changes and can be accessioned.
     deposit? && (new_cocina_object || status&.open?)
   end
+
+  def globus?
+    @globus ||= content.content_files.exists?(file_type: 'globus')
+  end
+
+  def endpoint_client_for(path)
+    GlobusClient::Endpoint.new(user_id: user.email_address, path:, notify_email: false)
```

Comment:
Does this class need to reach into the implementation details of the client or can it be rewritten to use the public interface we've set up on the `GlobusClient` class? I fear this is a slippery slope to the SdrClient situation we're still trying to get out of.